themes/10up-theme/tailwind.config.js

```diff
@@ -44,9 +96,10 @@ module.exports = {
 		},
 	},
 	plugins: [
-		/* Extend with 3rd party tailwind plugins or locally import your own */
-
-		/* Add Tailwind Prose https://tailwindcss.com/docs/typography-plugin */
-		tailwindProse(),
+		/*
+			Extend with 3rd party tailwind plugins or locally import your own
+			All plugins are included by default since they are not output if unused.
```

Comment:
@iansvo can we add a comment here explaining the use of prose in typographyPlugin?

Reply:
@Nikki-Jones Are you thinking something like the comment that was there (sorry didn't quite mean to just nuke it) and then perhaps a link to the relevant portion of the README?

Reply:
@iansvo sounds great!